lib/oci_utils/migrate/__init__.py

```diff
@@ -184,7 +184,7 @@ def get_config_data(key):
     """
     try:
         with OciMigrateConfParam(_oci_migrate_conf_file, key) as config:
-            return config.values()
+            return list(config.values())
```

Comment:
the unconditional conversion to a list causes results (and its types) from reading the configuration file are invalid and cause the program to terminate with an error.

Reply:
Reading this https://stackoverflow.com/questions/17695456/why-does-python-3-need-dict-items-to-be-wrapped-with-list
what yaml.load() is not alwasy the same anyway 
I guess that 2to3 has been confused by the method name: "value()"
What we can do is to rename this method , run the tool again 
see if things get better

Reply:
this is confirmed, changing the name to get_values() . did the trick

Reply:
I've pushed a new changeset to correct this

Reply:
tx.. started to ring a bell earlier today .. .